ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -13,10 +13,8 @@
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.HomePage;
-
```

Comment:
This space is needed for code quality and readability

Reply:
The space here is not neccessary ,  i guess

Reply:
According to our community import conventions, `devs` are encouraged to separate different categories of imports by a one space line between them. Look through other existing classes on how imports are organized

Reply:
Uhmm whats the difference between these  two 
**import org.openmrs.reference.groups.BuildTests;
import org.openmrs.reference.page.HomePage;**

public void loginTest(){}
And 
import org.openmrs.reference.groups.BuildTests;

**import org.openmrs.reference.page.HomePage;
public void loginTest(){
}** 
which of these looks smarter

Reply:
Have not got you well. But my reference was at line number 16

Reply:
ok thanks